src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  display: block;
+  width: 200px;
+  padding: 16px;
+  padding-top: 32px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  color: #060b35;
+  background-color: #fff;
+}
+
+.product-card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__item-name {
+  margin-bottom: 4px;
+
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.product-card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 23px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.product-card__value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.product-card__teg {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 140%;
+  color: #616070;
+}
+
+.product-card__buy-button {
+  height: 40px;
+  width: 100%;
+
+  border: 1px solid transparent;
+  background: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+
+  appearance: none;
```

Comment:
mdn [says,](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance#:~:text=Note%3A%20If,the%20newest%20browsers.) you should be careful with appearance property. I don`t fully figure out what it is :D